integration-tests/redis_test.go

```diff
@@ -139,11 +198,11 @@ var _ = Describe("Redis", Label("Redis"), func() {
 					HaveKey("instance_name"),
 					HaveKeyWithValue("labels", HaveKeyWithValue("pcf-instance-id", instanceID)),
 					HaveKeyWithValue("region", "us-west-2"),
-					HaveKeyWithValue("cache_size", BeNumerically("==", 2)),
+					HaveKeyWithValue("cache_size", BeNumerically("==", -1)),
```

Comment:
thoughts: this is because the new computed property but I believe we do not have to add it. We are cheating the test.

Reply:
Could you elaborate? An explanation on why cache_size is now -1 can be seen [here](https://github.com/cloudfoundry/csb-brokerpak-aws/pull/774/files#diff-63f5c7bf0664527e03a8d81e8860f43b44174ba17dbe21088e5aff9ee9230369L21).

Do you mean it would be better to just remove this line?
I wanted the test to reflect that cache_size still exists but it's taking the value "-1" when not explicitly specified.

Reply:
Fixed in commit: https://github.com/cloudfoundry/csb-brokerpak-aws/pull/774/commits/e1b98c701d764d72ef155faf667adf7868ceb771

Now cache_size is Nil when not passed in the plan.
Thanks for this feedback. Using a computed property with hardcoded value -1 felt very hacky.